BUILD

```diff
@@ -259,6 +262,25 @@ cc_proto_library(
 )
 
 ################################################################################
+# Well Known Types Proto Library Rules
+################################################################################
+
+internal_copied_filegroup(
+    name = "_internal_wkt_protos",
+    srcs = WELL_KNOWN_PROTOS,
+    dest = "",
+    strip_prefix = "src",
+    visibility = ["//visibility:hidden"],
+)
+
+[proto_library(
+    name = "well_known_types_" + proto[0] + "_proto",
```

Comment:
Is this prefix needed to avoid conflict? If not it seems better to just use the simpler name.

Reply:
We can also just use the simpler name. SGTM.

Reply:
Let's use the simpler version then. Can you update this pull request?